UKHomeOffice/sgmr-service
# SPCR-364 Move support email to an env variable

## Description
> We currently have the support email hard coded into the service. We should use environment variables as it makes it easier for us to change the email should it be updated.

Changes to `drone.yml` and `deployment.yml` should be checked carefully.

## To Test
- Make sure your local webpack config is pointing to the dev api
- `npm start`
-  It should compile successfully
- Go through the line changes in the code to see how the environment variables are passed
- Go to the accessibility statement and help pages
- You should see the support email
- Repeat the above process, running with Docker instead
 

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
